optimize/scalar/brent_min_test.go

```diff
@@ -0,0 +1,107 @@
+// Copyright ©2025 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package scalar
+
+import (
+	"math"
+	"testing"
+)
+
+func TestBrentMin(t *testing.T) {
+	tests := []struct {
+		name       string
+		f          func(float64) float64
+		min, max   float64
+		settings   *Settings
+		wantX      float64
+		wantStatus Status
+		distTol    float64 // Tolerance for the distance from wantX
+	}{
+		{
+			name: "Parabola: (x-2)^2 + 1",
```

Comment:
```suggestion
			name: "parabola",
```
The names don't need to fully describe the test, just be uniquely identifying. Similarly below.